pkg/models/purl_to_package.go

```diff
@@ -0,0 +1,50 @@
+package models
+
+import (
+	"github.com/package-url/packageurl-go"
+)
+
+var purlEcosystems = map[string]string{
+	"cargo":    "crates.io",
+	"deb":      "Debian",
+	"hex":      "Hex",
+	"golang":   "Go",
+	"maven":    "Maven",
+	"nuget":    "NuGet",
+	"npm":      "npm",
+	"composer": "Packagist",
+	"generic":  "OSS-Fuzz",
+	"pypi":     "PyPI",
+	"gem":      "RubyGems",
+}
```

Comment:
nit: could we make this a map of `models.Ecosystem` and start using the ecosystem constants?

(I think that would be ideal, but could result a lot more casting so might not be worth it)

Reply:
Changed this to use models.Ecosystem. I haven't changed `PackageInfo`'s `Ecosystem` field to be Ecosystem since that'll be a breaking change I believe.

Reply:
hmm ok I think you should include that as part of considering what to do going forward about ecosystem within the scanner per our meeting earlier